rfcs/require_webdriver_bidi.md

```diff
@@ -0,0 +1,114 @@
+# RFC 212: Introduce `require_webdriver_bidi` metadata tag
+
+## Summary
+
+To facilitate the gradual enablement of WebDriver BiDi transports, this RFC proposes the introduction of a new metadata tag, `require_webdriver_bidi` in html and js files in WPT tests. This metadata would allow for a gradual transition from [WebDriverProtocol](https://github.com/web-platform-tests/wpt/blob/9c76757f5332678f9952f6ccb3824f62d30eca1f/tools/wptrunner/wptrunner/executors/executorwebdriver.py#L644) to [WebDriverBidiProtocol](https://github.com/web-platform-tests/wpt/blob/9c76757f5332678f9952f6ccb3824f62d30eca1f/tools/wptrunner/wptrunner/executors/executorwebdriver.py#L736), ensuring that implementations can opt-in into WebDriver BiDi gradually in case there are implementation blockers like in Chrome's case. [Prototype](https://github.com/web-platform-tests/wpt/pull/48622).
```

Comment:
I believe this paragraph is poorly worded. What I meant is that the flag indicates if the test requires BiDi protocol. Currently if some specific testdriver API is not implemented in the implementation protocol and test tries to use it, a error **"{method name} is not implemented by testdriver-vendor.js"** is thrown. This part is kept untouched.

> Because one could imagine a test authored by people working on WebKit that works fine with BiDi there, but will fail in Chrome due to some BiDi implementation blocker. From the WebKit side, putting the require_webdriver_bidi metadata in the test seems fine, but from the Chrome side it doesn't, thus the test is the wrong place to store this metadata.

In this example the test will fail with the **"{method name} is not implemented by testdriver-vendor.js"** exception in Chrome.

@gsnedders please let me know if you think this approach is reasonable, and will update the RFC without mentioning "gradual enablement".

Reply:
It will be something like "`require_webdriver_bidi` is used to signal to the Browser/Executor that the test uses [testdriver.bidi](https://github.com/web-platform-tests/wpt/blob/2ab788df7f0d2ee35d67511348ce5eb4162c12d5/resources/testdriver.js#L55)".